src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -22,7 +22,12 @@ def __init__(self, parser: "ConfigParser"):
         "SCHEDULE_PREDICTION_FILE",
         "MULTFLT",
     ]
-    JUST_REMOVE_KEYWORDS = ["UMASK", "LOG_FILE", "LOG_LEVEL", "ENKF_RERUN"]
+    JUST_REMOVE_KEYWORDS = [
+        "UMASK",
+        "LOG_FILE",
+        "LOG_LEVEL",
+        "ENKF_RERUN",
```

Comment:
shouldn't `PLOT_SETTINGS` have an entry in this list?

Reply:
The keywords in that list is for those keywords we have nothing further t say about it than it can be removed.